.github/CONTRIBUTING.md

```diff
@@ -40,9 +40,24 @@ A bug must have an issue tracking it in the issue tracker that has been approved
 
 ## Contributing features
 
-We're happy to discuss schema proposals as long as they align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). 
+Thank you for wanting to contribute to Adaptive Cards. If you are planning to make changes to the Core SDK, please make sure these changes align with our [Core Design Principles](http://adaptivecards.io/documentation/#about-overview). Prior to writing any code that affects rendering, please make sure you follow the following steps:
+1.	Open a “Feature Request” issue to start the discussion on the feature you wish to see in the product. Please make sure you search our planned work to see if your feature is already planned. Please make sure you search our [planned work](https://portal.productboard.com/adaptivecards/1-adaptive-cards-features/tabs/6-planned) to see if your feature is already planned.
```

Comment:
NIT: looks like you copy + pasted from word, which used unicode quotes instead of typed quotes (note the curly-ness). It's a good idea to use the typed quotes instead of the word ones as they're more portable

Reply:
Awesome tip that I was unaware of, thanks for sharing!